File: frontend/src/app/interfaces/response/inventory-day.response.ts
export interface InventoryDayResponse {
  id: number;
  roomTypeId: number;
  roomTypeCode: string;
  roomTypeName: string;

  hotelId: number;
  hotelCode: string;
  hotelName: string;

  date: string; // yyyy-MM-dd
  totalUnits: number;
  availableUnits: number;
  price?: number;
  currency: string;
  status: string;
}

File: backend/src/main/java/com/fdsa/altairis/dto/response/InventoryDayResponse.java
package com.fdsa.altairis.dto.response;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

import java.math.BigDecimal;
import java.time.LocalDate;

@Getter
@Setter
@ToString
public class InventoryDayResponse {

    private Long id;
    private Long roomTypeId;
    private String roomTypeCode;
    private String roomTypeName;

    private Long hotelId;
    private String hotelCode;
    private String hotelName;

    private LocalDate date;
    private Integer totalUnits;
    private Integer availableUnits;
    private BigDecimal price;
    private String currency;
    private String status;

}
